test/python/qasm3/test_export.py

```diff
@@ -430,14 +430,19 @@ def test_multiple_pauli_evolution_gates(self):
 gate PauliEvolution(_t_0_) _gate_q_0, _gate_q_1 {
   rxx(2.0*_t_0_) _gate_q_0, _gate_q_1;
 }
+gate sxdg _gate_q_0 {
+  s _gate_q_0;
+  h _gate_q_0;
+  s _gate_q_0;
+}
 gate ryy(p0) _gate_q_0, _gate_q_1 {
-  rx(pi/2) _gate_q_0;
-  rx(pi/2) _gate_q_1;
+  sxdg _gate_q_0;
+  sxdg _gate_q_1;
   cx _gate_q_0, _gate_q_1;
   rz(p0) _gate_q_1;
   cx _gate_q_0, _gate_q_1;
-  rx(-pi/2) _gate_q_0;
-  rx(-pi/2) _gate_q_1;
+  sx _gate_q_0;
+  sx _gate_q_1;
```

Comment:
@Cryoris, I am wondering if the change for the standard definition of the RYYGate may have undesired/unexpected effects for pauli evolution synthesis, what do you think?